README.md

```diff
@@ -42,11 +42,22 @@ Circuitry.config do |c|
     HoneyBadger.flush
   end
   c.lock_strategy = Circuitry::Locks::Redis.new(url: 'redis://localhost:6379')
-  c.publish_async_strategy = :batch
-  c.subscribe_async_strategy = :thread
+  c.async_strategy = :thread
   c.on_thread_exit = proc { Mongoid.disconnect_sessions }
   c.on_fork_exit = proc { Mongoid.disconnect_sessions }
```

Comment:
We should be able to unify these into a single `on_async_exit` setting, one for publisher and one for subscriber.